src/deluge/gui/views/view.cpp

```diff
@@ -1966,6 +1966,7 @@ void View::drawOutputNameFromDetails(OutputType outputType, int32_t channel, int
 
 	if (display->haveOLED()) {
 		deluge::hid::display::oled_canvas::Canvas& canvas = hid::display::OLED::main;
+		deluge::hid::display::OLED::stopScrollingAnimation();
```

Comment:
why are some of these using stop scrolling animation plus clear() and some using clearMainImage() instead?

Reply:
because I realized later that I should just switch them all back and missed some